package/corrade/PKGBUILD

```diff
@@ -0,0 +1,50 @@
+# Author: williamjcm <w.jcm59@gmail.com>
+# Contributor: mosra <mosra@centrum.cz>
+# Based on the Corrade Arch Linux PKGBUILD maintained by xyproto and the MSYS2/MinGW PKGBUILD templates
+
+_realname=corrade
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=2018.10
+pkgrel=1
+pkgdesc='C++11/C++14 multiplatform utility library'
+provides=('corrade')
+arch=('any')
+url='https://magnum.graphics/corrade/'
+license=('MIT')
+makedepends=("${MINGW_PACKAGE_PREFIX}-cmake" 'git' "${MINGW_PACKAGE_PREFIX}-ninja")
+source=("${_realname}"::"git+https://github.com/mosra/corrade.git#tag=v$pkgver")
+sha1sums=('SKIP')
```

Comment:
```sh
source=("https://github.com/mosra/corrade/archive/v${pkgver}.tar.gz")
sha1sums=('240c68bfe89a0cc86411b6854358b1607c06d90d')
```

Not sure why the original package was using git, that's much slower. Also no need for `git` in makedepends, then.